recipes/install.rb

```diff
@@ -267,6 +267,13 @@
               'maximumpoolsize' => 400,
               'taskqueuecapacity' => 20000,
               'description' => 'Hopsworks Conda Executor Service'
+          },
+          'concurrent/jupyterExecutorService' => {
```

Comment:
This is problematic during the upgrade. When an upgrade runs this configuration doesn't get applied. You need to create the pool also in the default recipe.

Reply:
done